framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -173,6 +173,10 @@
     /** An instance of the {@link ContractsUtils} helper class. */
     protected final ContractsUtils contractsUtils;
 
+    /**
+     * A set of trees that are annotated with {@code @SuppressWarnings}/
+     *//*package-private*/ final List<Tree> treesWithSuppressWarnings;
```

Comment:
The formatting here looks a bit strange... not sure if there is something you can do to fix.